conjur-cli.gemspec

```diff
@@ -3,11 +3,11 @@ require File.expand_path('../lib/conjur/version', __FILE__)
 require "English"
 
 Gem::Specification.new do |gem|
-  gem.authors       = ["Rafal Rzepecki", "Kevin Gilpin"]
-  gem.email         = ["rafal@conjur.net", "kgilpin@conjur.net",]
+  gem.authors       = ["Conjur Maintainers"]
+  gem.email         = ["conj_maintainers@cyberark.com",]
   gem.summary       = %q{Conjur command line interface}
-  gem.homepage      = "https://github.com/conjurinc/cli-ruby"
-  gem.license       = 'MIT'
+  gem.homepage      = "https://github.com/cyberark/conjur-cli"
+  gem.license       = 'Apache 2.0'
```

Comment:
Don't we want an issue for that? to explain this change in licensing to the community?

Reply:
the licensing actually didn't change - it's been Apache 2.0 for a long time, we just had it wrong in the gemspec :(